cmd/process-agent/main_common.go

```diff
@@ -128,6 +130,15 @@ func runAgent(exit chan struct{}) {
 	tagger.Init()
 	defer tagger.Stop() //nolint:errcheck
 
+	// Set the log level
+	if opts.logLevel != "" {
+		if err := ddconfig.ChangeLogLevel(opts.logLevel); err != nil {
```

Comment:
nit: don't need := to assign err here. Since err already exists in the function, this will create a local variable inside the if statement